### PR TITLE
ESC 1030 , ESC-1025, ESC-1024, ESC-1005, ESC-968 - update various con…

### PR DESCRIPTION
**Pull request by masahmad**
various content updates

### CODE REVIEW

File: test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyControllerSpec.scala [hunk 1]
```diff
@@ -242,6 +242,26 @@ class SubsidyControllerSpec
...
+            mockRetrieveUndertaking(eori1)(undertaking.some.toFuture)
+            mockGet[SubsidyJourney](eori1)(Right(subsidyJourney.some))
+            mockTimeProviderToday(fixedDate)
+          }
+          val result = performAction("day" -> updatedDate.day, "month" -> updatedDate.month, "year" -> updatedDate.year)
+          val document = Jsoup.parse(contentAsString(result))
+
+          status(result) shouldBe BAD_REQUEST
+          val findElement = document.getElementById("claim-date-error").text()
+          findElement.replace("Error: ", "") shouldBe messageFromMessageKey("add-claim-date.error.incorrect-format")
```
Comment (habeeb01) [inline test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyControllerSpec.scala:259]: can we do `findElement shouldBe "Error: " + messageFromMessageKey("add-claim-date.error.incorrect-format")` instead, no need to replace "Error: " as it is part of the title when we have an error

Comment (masahmad) [reply test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyControllerSpec.scala]: updated

**Review by habeeb01 — APPROVED** [review]
LGTM 👍

**Review by pbyrne84 — APPROVED** [review]
Noice

**Pull request merged**